### ISSUE TITLE
Installed.  Now what?

### ISSUE THREAD

**Issue by stone212** [Chocobozzz/PeerTube#1786]
I read the FAQ https://github.com/Chocobozzz/PeerTube/blob/develop/FAQ.md

**What happened?**
I followed the instructions and PeerTube installed

**What do you expect to happen instead?**
I expected the UI or a console message would explain what to do now.  For example, what are "accounts" and how do I create one?  How do I upload content to my PeerTube server?

**Steps to reproduce:**

1. Install PeerTube

2. Go to your new instance at peertube.mysite.com

3. Wonder what to do next.

Basically there is a doc I am not seeing right?

**Comment by Chocobozzz** [comment]
You need to login with `root`: https://github.com/Chocobozzz/PeerTube/blob/develop/support/doc/production.md#administrator

You have some user documentation: https://docs.joinpeertube.org/lang/en/userdocs/

**Comment by stone212** [comment]
@Chocobozzz That helps.  Docs are not streamlined so I did not see that but it is what I needed.
